addons/mail/static/src/core/common/message_reaction_list.js

```diff
@@ -98,4 +98,10 @@ export class MessageReactionList extends Component {
             this.props.openReactionMenu();
         }
     }
+
+    onClickReactionList(reaction) {
+        this.preview.isOpen = false;
+        this.hover.setHover(false); // update the hover state of the preview
+        this.props.openReactionMenu(reaction);
+    }
```

Comment:
This doesn't look right to expose `setHover()` here.

I get it solves an issue when mouse-hovering the reaction again, because the useHover state is wrong. But the fix should be handled internally of the hook, not the business code to invoke a internal function of hook to solve the problematic internal state of hook.